### ISSUE TITLE
remember window size

### ISSUE THREAD

**Issue by stefan0uh** [ifedapoolarewaju/igdm#739]
the programm should remember the window size and maybe the position

**Comment by cgentry1972** [comment]
> the programm should remember the window size and maybe the position

Yes please fix this to where it remembers the window size because it opens nearly in full screen every time.

**Comment by DeviPirts** [comment]
Same issue here. Please fix the window size!
